components/salesforce_rest_api/actions/add-contact-to-campaign/add-contact-to-campaign.mjs

```diff
@@ -0,0 +1,50 @@
+import salesForceRestApi from "../../salesforce_rest_api.app.mjs";
+import {
+  removeNullEntries, toSingleLineString,
+} from "../../common/utils.mjs";
+
+export default {
+  key: "salesforce_rest_api-add-contact-to-campaign",
+  name: "Add Contact to Campaign",
+  description: toSingleLineString(`
+    Adds an existing contact to an existing campaign.
+    See [Event SObject](https://developer.salesforce.com/docs/atlas.en-us.228.0.object_reference.meta/object_reference/sforce_api_objects_campaignmember.htm)
+    and [Create Record](https://developer.salesforce.com/docs/atlas.en-us.228.0.api_rest.meta/api_rest/dome_sobject_create.htm)
+  `),
+  version: "0.0.1",
+  type: "action",
+  props: {
+    salesForceRestApi,
+    CampaignId: {
+      propDefinition: [
+        salesForceRestApi,
+        "sobjectId",
+        () => ({
+          objectType: "Campaign",
```

Comment:
Optional:
You can have these variables as constants.

Reply:
Kindly help clarify this. This field actually pulls from an endpoint